### DISCUSSION TITLE
href tags not working phalcon

### DISCUSSION THREAD

**Discussion by Jayden1411** [phalcon/cphalcon#15611] [General]
I am new to the framework.
I setup an new page but it can not go to the new page.

<h1>Hello!</h1>

<a href="signup">Sign Up Here!</a>    gives this:  **http://localhost/signup** and 404 not found
<a href="/signup">Sign Up Here!</a> gives this:  **http://localhost/jj/signup** and remains on same page

I am not sure how to fix this. Tried looking for solution online but cant break thru
SignupController is there with the action method as well as signup folder in views and its index.phtml

**Comment by niden** [comment]
Which project is this? Something you created? A sample application?
